CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
+## [3.40.1] - 2023-01-12
```

Comment:
Please simply use `## [develop]` as we don't know yet which version this will be in, an when it will be released.
```suggestion
## [develop]
```